### PR TITLE
feat(Spinner): consumed Penta tokens

### PR DESCRIPTION
**Pull request by thatblindgeye**
Work towards #5738

### CODE REVIEW

File: src/patternfly/components/Spinner/spinner.scss [hunk 1]
```diff
@@ -1,23 +1,34 @@
...
   // sizes
-  --#{$spinner}--m-sm--diameter: var(--#{$pf-global}--icon--FontSize--sm);
-  --#{$spinner}--m-md--diameter: var(--#{$pf-global}--icon--FontSize--md);
-  --#{$spinner}--m-lg--diameter: var(--#{$pf-global}--icon--FontSize--lg);
-  --#{$spinner}--m-xl--diameter: var(--#{$pf-global}--icon--FontSize--xl);
-
+  --#{$spinner}--m-sm--diameter: var(--pf-t--global--icon--size--md);
+  --#{$spinner}--m-md--diameter: var(--pf-t--global--icon--size--lg);
+  --#{$spinner}--m-lg--diameter: var(--pf-t--global--icon--size--xl);
+  --#{$spinner}--m-xl--diameter: var(--pf-t--global--icon--size--2xl);
```
Comment (thatblindgeye) [inline src/patternfly/components/Spinner/spinner.scss:21]: Do we still need this modifier class since this is the default size of a spinner?

Comment (srambach) [reply src/patternfly/components/Spinner/spinner.scss:21]: I lean to the side of leaving the modifier for completeness but I wouldn't fight about it.

**Review by srambach — APPROVED** [review]
🍭

File: src/patternfly/components/Spinner/spinner.scss [hunk 2]
```diff
@@ -1,23 +1,34 @@
 // @debug $spinner; // check your variable names located in src/patternfly/sass-utilities/component-namespaces.scss
 
+:root,
 .#{$spinner} {
-  --#{$spinner}--diameter: var(--#{$pf-global}--icon--FontSize--xl);
+  --#{$spinner}--diameter: var(--pf-t--global--icon--size--2xl);
   --#{$spinner}--Width: var(--#{$spinner}--diameter);
   --#{$spinner}--Height: var(--#{$spinner}--diameter);
-  --#{$spinner}--Color: var(--#{$pf-global}--primary-color--100);
+  --#{$spinner}--Color: var(--pf-t--global--icon--color--brand--default);
```
Comment (mcoker) [inline src/patternfly/components/Spinner/spinner.scss:8]: This seems like the right value but in figma it's defined as the border--color--brand token. cc @lboehling @andrew-ronaldson do you know which is right?

File: src/patternfly/components/Spinner/spinner.scss (hunk 1 above)
Comment (mcoker) [reply src/patternfly/components/Spinner/spinner.scss:21]: IMO the "inline" spinner should be the default, then you would opt-in to a size variation. It seems odd that the default spinner is so large.

**Review by mcoker — APPROVED** [review]

**Pull request merged**